src/Utils/OccRunner.php

```diff
@@ -115,16 +115,24 @@ public function runJson($command, $args = []){
 	protected function runAsRequest($command, $args){
 		$application = $this->getApplication();
 		$client = new Client();
+		$endpointBase = $application->getEndpoint();
+		$params = [
+			'timeout' => 0,
+			'json' => [
+				'token' => $application->getAuthToken(),
+				'params'=> $args
+			]
+		];
+		
+		// Skip SSL validation for localhost only as localhost never has a valid cert
+		if (preg_match('^https:\/\/localhost\/.*/i'), $endpointBase){
```

Comment:
@PVince81 `localhost` is used as a last fallback host, so I don't think so. 
But trying 127.0.0.1 instead of localhost looks sane. I need to think it over...